companies/costco.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I'm not sure how the Costco file got stuck in this PR as well :( @Nimanita do you think you could delete the file from this branch?